### ISSUE TITLE
whish to merge the code with [skalidindi3/kicad-colors](https://github.com/skalidindi3/kicad-colors)

### ISSUE THREAD

**Issue by easyw** [pointhi/kicad-color-schemes#11]
wish to merge the code with https://github.com/skalidindi3/kicad-colors

**Comment by poeschlr** [comment]
It seems the owner of the other repo is not really interested in merging the two. They reason it with "other projects also have a split up community for colour schemes."

**Comment by Pun-e** [comment]
The other repo hasn't had a "real" commit in over 2 years, but does have some really nice schemes like onedark, nord and rebecca.

Is there any chance we could have their open-source schemes rolled into the one repo, and hopefully get this unified theme database pushed into KiCad in time so that all users would have access to them?

**Comment by dmitchellspace** [comment]
If someone wants to give me access to this repository, I have a local commit which merges in the other themes, plus does a bunch of other things - including displaying a list of available themes for the user to interactively pick from and correctly making/saving a backup eeschema.

**Comment by pointhi** [comment]
Please create Pull Requests for all those things you want to have merged.

**Comment by dmitchellspace** [comment]
Certainly, but I am unable to push my branch. I'm getting authentication errors which suggests I need additional repository permissions?

**Comment by pointhi** [comment]
You have to create a personal fork and push to this fork, not onto my repository.

**Comment by dmitchellspace** [comment]
@pointhi Done: https://github.com/pointhi/kicad-color-schemes/pull/22
